04_removeFromArray/removeFromArray.js

```diff
@@ -1,6 +1,16 @@
-const removeFromArray = function() {
-
+const removeFromArray = function (...values) {
+  /* This line declares function that can accept multiple arguments and puts them in an array */
+  const array =
+    values.shift(); /*Since the first passed value in values[] is an array that needs to be compared to values array, this needs to take it out of the values array */
+  const newArray = []; /* New array that is going to contain wanted values */
```

Comment:
Better naming would be:
```suggestion
  const filteredArray = []; /* New array that is going to contain wanted values */
```
```suggestion
  const newArray = []; /* New array that is going to contain wanted values */
```